playbook.yml

```diff
@@ -22,8 +22,9 @@
     when: sonar == "db"
 
   - include_role:
-      name: "{{ item }}"
+      name: geerlingguy.java
```

Comment:
add a comment with link to the bug you found

Reply:
to add the link in the playbook itself?

Reply:
yes